### PR TITLE
{bio}[intel/2017a] Mothur v1.39.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2690.swalot.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b3061dd48cb7bf8b159649bd59042ba8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
gligar01.gligar.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1fcda735f4f9985188b527482b53f937 for a full test report.

File: easybuild/easyconfigs/g/gzip/gzip-1.8-GCCcore-6.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'gzip'
+version = '1.8'
+
+homepage = 'http://www.gnu.org/software/gzip/'
+
+description = """
```
Comment (wpoely86) [inline easybuild/easyconfigs/g/gzip/gzip-1.8-GCCcore-6.3.0.eb]: fix formatting?

File: easybuild/easyconfigs/m/Mothur/Mothur-1.39.5_fix-hardcoding.patch [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
++MOTHUR_FILES=''
+ RELEASE_DATE = "\"3/20/2017\""
+ VERSION = "\"1.39.5\""
+ 
+@@ -31,13 +31,13 @@
+ 
+ # Fastest
+ ifeq  ($(strip $(OPTIMIZE)),yes)
+-    CXXFLAGS += -O3
++    CXXFLAGS += #-O3
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/Mothur/Mothur-1.39.5_fix-hardcoding.patch]: just don't add it?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
gligar01.gligar.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8027de436142cf30e368c202b113a23a for a full test report.

**Pull request merged**